build/org.eclipse.cdt.managedbuilder.core/src/org/eclipse/cdt/managedbuilder/internal/core/jsoncdb/generator/CompilationDatabaseGenerator.java

```diff
@@ -438,11 +438,11 @@ public boolean visit(IResourceProxy proxy) throws CoreException {
 
 	}
 
-	private String findCompilerInPath(ITool tool, IConfiguration config) {
+	private String findCompilerInPath(ITool tool, IConfiguration config, IManagedCommandLineInfo cmdLInfo) {
```

Comment:
I think the `toolMap` cache is now incorrect - the key should probably be tool + compiler name before it is expanded, or even just the compiler name before it is expanded